src/api/quantum/meta/Universal.java

```diff
@@ -0,0 +1,13 @@
+package api.quantum.meta;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Represents a universal object
+ */
+@Target({ElementType.TYPE, ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.PACKAGE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Universal {}
```

Comment:
What exactly does Universal represent? (is it for defining client and server side stuff, in which case why? You might need to improve documentation)